src/__tests__/tigris.filters.spec.ts

```diff
@@ -305,11 +329,20 @@ export interface IUser extends TigrisCollectionType {
 	balance: number;
 }
 
-export interface IUser1 extends TigrisCollectionType {
-	id: BigInt;
+@TigrisCollection("user1")
+export class IUser1 implements TigrisCollectionType {
+	@PrimaryKey({ order: 1 })
+	id: bigint;
+	@Field()
 	name: string;
+	@Field()
 	balance: number;
+	@Field()
 	isActive: boolean;
+	@Field(TigrisDataTypes.DATE_TIME)
+	createdAt: string;
+	@Field()
+	updatedAt: Date;
```

Comment:
`createdAt` and `updatedAt` demonstrate the two different ways to use timestamps